### PR TITLE
Fix Wine install steps from WineHQ.

### PR DESCRIPTION
**Pull request by Alexhuszagh**

### CODE REVIEW

**Comment by Alexhuszagh** [comment]
bors try --target  x86_64-pc-windows-gnu

File: docker/wine.sh [hunk 1]
```diff
@@ -14,9 +14,12 @@ main() {
     # add repository for latest wine version and install from source
     # hardcode version, since we might want to avoid a version later.
     wget -nc https://dl.winehq.org/wine-builds/winehq.key
-    mv winehq.key /usr/share/keyrings/winehq-archive.key
+    # workaround for wine server synchronization, see #1035
+    mkdir -p /etc/apt/keyrings
+    mv winehq.key /etc/apt/keyrings/winehq-archive.key
     wget -nc https://dl.winehq.org/wine-builds/ubuntu/dists/focal/winehq-focal.sources
     mv winehq-focal.sources /etc/apt/sources.list.d/
+    sed -i s@/usr/share/keyrings/@/etc/apt/keyrings/@ /etc/apt/sources.list.d/winehq-focal.sources || true
```
Comment (Emilgardis) [inline docker/wine.sh:22]: can you section this with some newlines to make it clear what the workaround is

Comment (Alexhuszagh) [reply docker/wine.sh]: Does that look better?

Comment (Emilgardis) [reply docker/wine.sh]: nice!

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**